packages/core/src/App/Containers/AcuityDownloadModal/acuity-download-modal.tsx

```diff
@@ -50,6 +59,25 @@ const AcuityDownloadModal = ({ is_acuity_modal_open, is_eu, setIsAcuityModalOpen
                         />
                     </Text>
                 </div>
+                <div className='acuity-download-modal__body--guide'>
+                    <Text as='p' line_height='m' align='center'>
+                        <Localize
+                            i18n_default_text='<1>Need help using Acuity?</1><0/>Check out this <2>user guide</2>.'
```

Comment:
`Check out this user guide` should be a separate `text`, 
size is different as well.